zuul.d/jobs.yaml

```diff
@@ -0,0 +1,10 @@
+---
+- job:
+    name: install-yaml-molecule
+    parent: cifmw-molecule-base
+    required-projects:
+      - openstack-k8s-operators/install_yamls
+      - github.com/openstack-k8s-operators/ci-framework
+      - github.com/openstack-k8s-operators/repo-setup
```

Comment:
I'm not sure we actually need that one... ? since this is the very project we're testing?

Reply:
nice :).
Maybe squash patches so that only one is actually pushed? Cleaner history in the end ;).